testing/test_pytester.py

```diff
@@ -36,23 +38,23 @@ class rep:
     failures = recorder.getfailures()
     assert failures == [rep]
 
-    class rep:
+    class rep2:
         excinfo = None
         passed = False
         failed = False
         skipped = True
         when = "call"
 
-    rep.passed = False
-    rep.skipped = True
-    recorder.hook.pytest_runtest_logreport(report=rep)
+    rep2.passed = False
+    rep2.skipped = True
+    recorder.hook.pytest_runtest_logreport(report=rep2)
```

Comment:
Diff churn like this makes me think that `--allow-redefinition` would be better ([docs](https://mypy.readthedocs.io/en/latest/command_line.html#cmdoption-mypy-allow-redefinition)).
If not for "src" then maybe for "testing" at least, and maybe only per module there then?

Reply:
I personally think this makes the code less confusing. I generally prefer to fix than to use `allow-redefinition`, for clarity and for the occasional errors it catches. Though it might be slightly annoying sometimes.